### ISSUE TITLE
Error: Cannot find module 'ext'

### ISSUE THREAD

**Issue by alexwall** [laktek/realie#3]
Hi
during the installation I had problems so I cant't start realie

git://gist.github.com/573607.git

**Comment by benoror** [comment]
I have the same prblem. Look at this: http://pastie.org/1149012

**Comment by markc** [comment]
Yep, same here too. It's because express tries to include class.js which doesn't seem to exist so the express submodule update fails which means ext is not available, and it just so happens ext is the first module that realie tries to require. Pity, I'd love to try realie out.

**Comment by alexwall** [comment]
I've fixed it. http://github.com/alexwall/realie

Clone my Repo git@github.com:alexwall/realie.git
